### PR TITLE
feat(content-utils): Add Git times getter for collection entries

### PR DESCRIPTION
**Pull request by Fryuni**

### CODE REVIEW

File: packages/content-utils/src/integration/gitTimePlugin.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	load(id, { ssr } = {}) {
+		if (id !== RESOLVED_MODULE_ID || !ssr) return;
+
+		return `
+import {setContentPath} from '@inox-tools/content-utils/runtime/git';
+import {getLatestCommitDate, getOldestCommitDate} from '@inox-tools/content-utils/runtime/liveGit';
+
+console.log('This is running');
+setContentPath(${JSON.stringify(contentPath)});
+console.log('This is also running');
```
Comment (BryceRussell) [inline packages/content-utils/src/integration/gitTimePlugin.ts:22]:
```suggestion
setContentPath(${JSON.stringify(contentPath)});
```

File: packages/content-utils/src/integration/gitTimePlugin.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	resolveId(id) {
+		if (id === MODULE_ID) return RESOLVED_MODULE_ID;
+	},
+	load(id, { ssr } = {}) {
+		if (id !== RESOLVED_MODULE_ID || !ssr) return;
+
+		liveGit.setContentPath(contentPath);
+		const trackedFiles = liveGit.getAllTrackedCommitDates();
+
+		console.log(trackedFiles);
```
Comment (BryceRussell) [inline packages/content-utils/src/integration/gitTimePlugin.ts:42]:
```suggestion
```

File: docs/src/content/docs/content-utils/git-time.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+   ```astro title="src/pages/blog/[...slug].astro"
+   ---
+   import { getOldestCommitDate } from '@it-astro:content/git';
+
+   const { slug } = Astro.params;
+
+   const entry = await getEntry('blog', slug);
+   const creationDate = await getOldestCommitDate('blog', slug);
+   ---
```
Comment (BryceRussell) [inline docs/src/content/docs/content-utils/git-time.mdx:41]: This example could also show that it is possible to do this: `await getOldestCommitDate(entry)`

**Review by BryceRussell — CHANGES_REQUESTED** [review]
I tried previewing this locally, but the example project is giving me this error:

> Cannot find module '@it-astro:content/git' imported from 'C:/Users/Bryce/Desktop/Projects/Astro/inox-tools/examples/content-injection/src/pages/index.astro'

**Comment by Fryuni** [comment]
> I tried previewing this locally, but the example project is giving me this error:

Maybe you haven't built the lib with the new code. Try running `pnpm -w build:examples`, it builds the dependencies before building the examples.

**Comment by BryceRussell** [comment]
I got it working :smile: I did build it, not sure why it broke because all I did was re-run my commands

**Review by BryceRussell — APPROVED** [review]
LGTM!

**Review by OliverSpeir — APPROVED** [review]
🎉

**Review by florian-lefebvre — APPROVED** [review]

**Pull request merged**